### PR TITLE
Nested AJAX form validation failure fix

### PR DESCRIPTION
**Pull request by jclemans**
## Summary ##
Adds a call to re-bind form field validation events after a nested AJAX form is loaded.

## Background ##
The way that _validator.js is set up runs through a loaded page and binds all the form fields that have `.data-validate` on them. But a nested form that is loaded via AJAX does not get addressed, so this change re-runs the binding action on the form after the new form fields are loaded.

Related issues:

- Redmine 75403
- Flow 35293743

### CODE REVIEW

File: app/assets/javascripts/fae/form/_ajax.js [hunk 1]
```diff
@@ -66,6 +66,9 @@ Fae.form.ajax = {
         $wrapper.find('.input.file').fileinputer();
       }
 
+      // Re-bind form validations to catch new AJAX loaded fields
+      Fae.form.validator.bindValidationEvents();
```
Comment (jamesmk) [inline app/assets/javascripts/fae/form/_ajax.js]: @jclemans We chatted IRT about adding the ability to optionally scope this method so we don't have to rebind all form event handlers every time nested form AJAX is called. Adding as a review item, let me know if you hit any walls on it.

thanks

File: app/models/fae/static_page.rb [hunk 2]
```diff
@@ -42,15 +42,15 @@ def self.setup_dynamic_singleton
           languages.each do |lang|
             # Save with suffix for form fields
             define_association("#{name}_#{lang}", type)
-            define_validations("#{name}_#{lang}", type, value[:validates]) if value.try(:[], :validates).present?
```
Comment (jclemans) [inline app/models/fae/static_page.rb]: @jamesmk - not sure why these changes static_page diffs are showing up. I just pulled from master to bring my branch up to date with your recent changes.

**Comment by jclemans** [comment]
@jamesmk - updated to fix the failing capybara tests / IE11 issue. Just the one flickering date_sort_spec failing now.

**Comment by jamesmk** [comment]
thanks @jclemans!
